analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -96,6 +99,8 @@ class GoMod(
 
     override fun transformVersion(output: String) = output.removePrefix("go version go").substringBefore(' ')
 
+    override fun getVersionRequirement(): RangesList = RangesListFactory.create("[1.19.0,)")
```

Comment:
If this is a requirement to make the code in the previous commit work, this code should be squashed into the previous commit (or be moved to a preceding commit to guarantee the requirements for the upcoming change).